test/lib/cql_test_env.cc

```diff
@@ -109,6 +110,51 @@ static future<> tst_init_ms_fd_gossiper(sharded<gms::feature_service>& features,
 }
 // END TODO
 
+class debug_fmt_consumer : public cql3::query_result_consumer {
```

Comment:
Let's call it cql_result_message_builder so what it does it more explicit. "debug" can mean many things, I expected it to print out the data when I saw the name.

Reply:
done.